css/two_column.css

```diff
@@ -0,0 +1,120 @@
+html { 
+    font-size: 15px; 
+} 
+
+@media (min-width: 1500px) { 
```

Comment:
The laptop case has been tested (by me and @Sfshaza ), and it works. The code samples are stacked instead of side-by-side. I'll check it again.

Making changes to align as close as possible w/bootstrap defaults. However, because this is a wider layout, the desktop needs a max-width of 1470px in order to avoid horizontal scrollbars on most of the left-hand code snippets.